dex/src/main/resources/application.conf

```diff
@@ -482,6 +482,18 @@ waves.dex {
   web-sockets {
     # Interval between messages that will be sent to subscribers
     messages-interval = 100ms
+
+    # Max connection lifetime before force stop
+    max-connection-lifetime = 24 hours
+
+    # Server side ping-pong settings
+    ping-pong-settings {
+      # Interval between ping messages sent to client
+      ping-interval = 10s
+
+      # Timeout for client response
+      pong-timeout = 30s
```

Comment:
`pong-timeout` is :ok: imho

Reply:
Then we need to update a comment:
```
# Timeout for expected client response
```
Because there is an example:
```
> ping1
< pong1
> ping2
> ping3
< pong2
The connection is closed
```
a client responded with a pong, but connection was close anyway.

Reply:
Done